.travis.yml

```diff
@@ -19,10 +26,6 @@ before_install:
  # Useful for debugging any issues with conda
  - conda info -a
 
- # create and activate the build/test environment
- - conda create -q -n tenv python=$TRAVIS_PYTHON_VERSION pip cmake
```

Comment:
For whatever reason, making and activating tenv was completely failing on osx, so I just switched it over to building in the root.  Since this isn't shared, I thought what the heck :)